pandas/core/resample.py

```diff
@@ -778,6 +778,11 @@ def _wrap_result(self, result):
         # convert if needed
         if self.kind == 'period' and not isinstance(result.index, PeriodIndex):
             result.index = result.index.to_period(self.freq)
+
+        if isinstance(result, com.ABCSeries) and result.empty:
```

Comment:
This should go at line 437 (the other ``_wrap_result``); this is for a sub-class DatetimeIndexResampler (and it calls the other one).